presto-tests/src/test/java/com/facebook/presto/memory/TestMemoryManager.java

```diff
@@ -243,6 +243,18 @@ public void testQueryMemoryLimit()
         }
     }
 
+    @Test(timeOut = 240_000, expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "Exceeded cpu limit of :.*")
```

Comment:
Test fails. The message here does not match the message actually thrown (colon). See Travis.